HW2/P2/P2.py

```diff
@@ -16,8 +16,8 @@
     # Generate some test data, first, uncorrelated
     ########################################
     orig_counts = np.arange(1000, dtype=np.int32)
-    src = np.random.randint(1000, size=1000000).astype(np.int32)
-    dest = np.random.randint(1000, size=1000000).astype(np.int32)
+    src = np.random.randint(1000, size=1000).astype(np.int32)
```

Comment:
Is there a reason you changed the size of the source and destination arrays? I left mine the same.